workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/learn-course-introduction/page-definition.json

```diff
@@ -32,80 +32,352 @@
 							"customCSS": ".[$FRAGMENT_CLASS$] {\nbackground: linear-gradient(91.07deg, #001742 0%, #0011A8 24.6%, #0B5FFF 49%, #0B5FFF 74.5%, #0B5FFF 100%),\nradial-gradient(26.56% 43.82% at 0% 0%, #00256B 20.83%, rgba(0, 37, 107, 0) 100%) /* warning: gradient uses a rotation that is not supported by CSS and may not behave as expected */,\nradial-gradient(14.55% 34.6% at 100% 34.6%, rgba(55, 124, 255, 0.68) 0%, rgba(55, 124, 255, 0) 100%);\n\n}",
 							"fragmentStyle": {
 								"borderRadius": "borderRadiusLg",
-								"height": "216px",
-								"paddingBottom": "4",
-								"paddingLeft": "4",
-								"paddingRight": "4",
-								"paddingTop": "4"
+								"height": "216px"
 							},
 							"indexed": true,
 							"layout": {
-								"paddingBottom": 2,
-								"paddingLeft": 2,
-								"paddingRight": 2,
-								"paddingTop": 2
+								"contentDisplay": "FlexRow",
+								"justify": "SpaceBetween"
 							},
 							"name": "Header Banner"
 						},
 						"pageElements": [
 							{
 								"definition": {
-									"fragment": {
-										"key": "dialect-text",
-										"siteKey": "Liferay Learn"
+									"fragmentStyle": {
+										"paddingBottom": "4",
+										"paddingLeft": "4",
+										"paddingRight": "4",
+										"paddingTop": "4"
 									},
-									"fragmentConfig": {
-										"textType": "h5",
-										"truncateText": false
+									"indexed": true,
+									"layout": {
+										"paddingBottom": 2,
+										"paddingLeft": 2,
+										"paddingRight": 2,
+										"paddingTop": 2
 									},
-									"fragmentFields": [
-										{
-											"id": "element-text",
-											"value": {
-												"fragmentLink": {
+									"name": "Header Content"
+								},
+								"pageElements": [
+									{
+										"definition": {
+											"fragmentStyle": {
+												"backgroundColor": "colorNeutral1",
+												"borderRadius": "50rem",
+												"height": "40px",
+												"marginBottom": "2rem",
+												"width": "116px"
+											},
+											"indexed": true,
+											"layout": {
+												"align": "Center",
+												"contentDisplay": "FlexRow",
+												"justify": "Center"
+											},
+											"name": "Course Tag"
+										},
+										"pageElements": [
+											{
+												"definition": {
+													"fragment": {
+														"key": "BASIC_COMPONENT-image"
+													},
+													"fragmentConfig": {
+														"imageSize": "w-0"
+													},
+													"fragmentFields": [
+														{
+															"id": "image-square",
+															"value": {
+																"fragmentImage": {
+																	"description": {
+																		"value_i18n": {
+																			"en_US": ""
+																		}
+																	},
+																	"url": {
+																		"value_i18n": {
+																			"en_US": "/documents/d/liferay-learn/education_courses_blue-svg?imagePreview=1"
```

Comment:
Use placeholder instead absolute path

Reply:
will be done in the icons/images refactoring ticket